tests/ui/nll/user-annotations/dump-fn-method.rs

```diff
@@ -1,8 +1,6 @@
 // Unit test for the "user substitutions" that are annotated on each
 // node.
 
-// compile-flags:-Zverbose
```

Comment:
This test's output changes with the modification to how bound/placeholder tys print under `-Zverbose`, though arguably I could revert that change -- just made debug a bit easier for me.